plugin/buycourses/src/invoice.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/* For license terms, see /license.txt */
+
+/**
+ * Print invoice of the Buy Courses plugin.
+ *
+ * @package chamilo.plugin.buycourses
+ */
+use Chamilo\CoreBundle\Component\Utils\ChamiloApi;
```

Comment:
Poner use Chamilo\CoreBundle\Component\Utils\ChamiloApi;
Después de /* For license terms, see /license.txt */